.evergreen.yml

```diff
@@ -33,8 +33,8 @@ functions:
     - command: git.get_project
       params:
         directory: src
-    - &generate-compass-env
```

Comment:
Isn't this needed?

Reply:
The command is needed, but defining it as a re-usable part of the template is not, unless I'm missing something. It's used only in `restore` and `test-plugin` and both of those parts of config are removed in this PR